### ISSUE TITLE
Getting error with one of the models only on smartphones

### ISSUE THREAD

**Issue by dhruvadhia1** [google/model-viewer#2155]
The configurator keeps loading repeatedly only on mobile phones and crashes ultimately with this model. However, the same model works on mobile via babylonjs. https://thecela.com/experiment/elder/

#### Live Demo
https://tinyurl.com/yec6vpaz 

### Version
Latest 

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [x] Safari

### OS

- [x] Android
- [x] iOS

**Comment by elalish** [comment]
Is this just a problem with one model? Does it repro with a simple model-viewer element? It looks like a very large model, so you might be running out of GPU memory on the phone.

**Comment by dhruvadhia1** [comment]
I tested it on simple model-viewer one line of code and it doesnt work. Now its interesting to note that the model viewer is working with some really high poly models with more than a million polys, but with this one it has issues. 

Here is a test with minimal model viewer code. And it crashes only on mobile. 
https://thecela.com/experiment/elder/test.html

But what I found interesting was it crashes only when height and width are at 100% for modelviewer in css. So somehow the larger paint area is causing the crash? And we hardly have this with any other models.
